.ipynb_checkpoints/Untitled-checkpoint.ipynb

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Normally we don't store checkpoints on github. It's kind of double up.  Github stores commitpoints, and having checkpoint within commit points is unnecessary.

Please add to you .gitignore

**scripts/.ipynb_checkpoints/**

or wherever you have your notebook files